ca/server.go

```diff
@@ -36,6 +36,10 @@ type Server struct {
 	joinTokens                  *api.JoinTokens
 	reconciliationRetryInterval time.Duration
 
+	// skipAutoUpdateRootCA, if set to true, causes the CA server to skip automatically updating
+	// the signing credentials based on changes in the memory store
+	skipAutoUpdateRootCA bool
```

Comment:
I'll trust your judgement on this. I didn't like having an option that's never enabled by the actual program, but it's basically enabling or disabling a function call, and the alternative is having the test harness do significantly more.